inmysparetime.html

```diff
@@ -1,12 +1,19 @@
 <!DOCTYPE html>
 
 <html lang="en-UK">
-
 <head>
-		<title>What I Do In my Spare Time... </title>
-	<link href="bootstrap-3.3.6-dist/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
-	<link rel='stylesheet' style='text/css' href='CSS.css'>
-
+	<meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+	
+    <title>Hobbies and Interests</title>
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
```

Comment:
It's generally preferable to use HTTPS urls where you can, and MaxCDN files are usually available via HTTPS, so this line should probably be `https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css`